File: 2 -- Arrays/Reverse_Array.cpp
#include <iostream>
using namespace std;

int main(){

    int arr[]={2,4,6,8};
    int size=4;

    for(int i=0;i<size;i++){
        cout<<arr[i]<<" ";
    }
    cout<<endl;

    for(int i=0;i<size;i++){
        for(int j=size-1;i<=j;j--){
            int temp=arr[i];
            arr[i]=arr[j];
            arr[j]=temp;
            i++;
        }
    }

    for(int i=0;i<size;i++){
        cout<<arr[i]<<" ";
    }
}

File: 2 - Arrays/1D Arr/reversearray.java
public class reversearray{
    public static void main(String[] args) {
        int arr[]={1,2,3,4,5};
        int start=0;
        int end=arr.length-1;
        int temp;
        while(start<end){
            temp=arr[start];
            arr[start]=arr[end];
            arr[end]=temp;
            start++;
            end--;
        }
        for(int k=0;k<arr.length;k++){
            System.out.print(arr[k]);
        }
    }
}